src/data/pages.json

```
@@ -885,6 +885,10 @@
                   {
                     "title": "Blocked accounts",
                     "id": "blocked-accounts"
+                  },
+                  {
+                    "title": "Issues receiving reviewer emails",
```

```suggestion
                    "title": "Issues receiving emails from AMO",
```
Or something around that. It's not just about reviewer emails.